### ISSUE TITLE
[Meneur] Proposition de nouvelles commandes 

### ISSUE THREAD

**Issue by HemnesLGeL** [Lesterpig/openparty-mafia#28]
J'ouvre cette issue pour les idées de nouvelles commandes pour le MDJ. Je commence :
- [ ]  `/revive <player> <role>` : ressucite un joueur mort, en lui attribuant un nouveau rôle.
- [ ] `/mute <player>` : empêche un joueur de parler
- [x] `/save <player>` : sauve un joueur de la mort
- [x] `/murder <player>`
- ~~`/mix` : mélange les cartes, comme si une nouvelle partie démarrait~~

**Comment by Lesterpig** [comment]
Je suis d'accord pour les trois premières, mais pourquoi la dernière ? En gros c'est un "restart" ?

**Comment by HemnesLGeL** [comment]
C'est pour pimenter la partie, quand tu penses avoir compris avec qui tu es allié, on change tous les rôles ! :D
(je suis d'accord que c'est pas prioritaire mais l'idée me plait :p)

**Comment by Lesterpig** [comment]
Hm, à voir. Techniquement, ça risque de poser des problèmes :8
Je m'occupe de quelques commandes demain je pense :)
